README.md

```diff
@@ -1,56 +1,35 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TO DO LIST
 
-> Description the project.
+> This is a Website that lets you add, remove, update and delete tasks. It utilizes local storage to store your tasks
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML
+- CSS
+- JavaScript
+- Webpack
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+* Click the Code button in the top right corner
+* Download ZIP
+* Unzip the file: to-do-list.zip
+* Open index.html with your web browser
+* (Optional) You should have Node.js for work on the application running properly.
 
```

Comment:
[OPTIONAL] Thanks for giving us a simple way to get your code. But I think it would be nice to also include the more technical way like `git clone ...  cd ...` for the more nerdy folks like us 😃